src/components/tree-views-container.js

```diff
@@ -495,16 +496,17 @@ const TreeViewsContainer = () => {
         return () => {
             wsToClose.close();
         };
-    }, []);
+    }, [dispatch]);
```

Comment:
there is no calls to dispatch in here... remove this dependency ? Then the effect will be run only on componentDidMount ?